kuma/api/v1/search/__init__.py

```diff
@@ -153,10 +153,13 @@ def _find(params, total_only=False, make_suggestions=False, min_suggestion_score
             # This can happen if the search happened right as the index had
             # just been deleted due to a fresh re-indexing happening in Yari.
             exceptions.NotFoundError,
+            # This can happen when the index simply isn't ready yet.
+            exceptions.TransportError,
```

Comment:
I've seen this happen quite a lot when I do my manual testing. It's not unrealistic that you get:
```
elasticsearch.exceptions.TransportError: TransportError(503, 'search_phase_execution_exception')
```
that happens when index isn't ready. There's some window of time between when the *server* is ready to accept connections and the *index* being ready. 
I've seen it many times that you can just wait a little, and refresh and the problem goes away. So I added it to the list of retry exceptions. 

Yes, this is unrelated to the (original) issue at hand.